watchfaces/000-default-digital.qml

```diff
@@ -98,25 +99,24 @@ Item {
             var ctx = getContext("2d")
 
             ctx.font = "25 " + height/15 + "px Raleway"
-            ctx.fillText(wallClock.time.toLocaleString(Qt.locale(), "AP"), width*0.85, height*0.4);
+            ctx.fillText(wallClock.time.toLocaleString(Qt.locale("en_EN"), "AP"), width*0.894, height*0.371);
```

Comment:
Why did you force "en_EN" ? If this is required to actually show AM/PM when a language different than English is selected, you should use the same trick in
003-alternative-digital-2.qml
005-alternative-scifi.qml

Reply:
I tried alot and many languages give awkward long words instead of am/pm 
Figured it is most consitant to force en_EN from design POV.
For scifi it was not pressing that much because the international am/pm words have space to flow.
In digital-2 i forgot to update the findings i had during making funky-town as i did in default-digital. 
Will change it to consitant en_EN enforcement.